services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImplV2.java

```diff
@@ -428,7 +437,16 @@ public boolean canArchiveNews(Identity currentIdentity, String newsAuthor) {
   }
 
   private News createDraftArticleForNewPage(News draftArticle, String pageOwnerId, String draftArticleCreator) throws Exception {
-    Page articlesRootPage = getArticlesRootPage(pageOwnerId);
+    Wiki wiki = wikiService.getWikiByTypeAndOwner(WikiType.GROUP.name().toLowerCase(), pageOwnerId);
+    Page articlesRootPage = null;
+    if (wiki != null) {
+      articlesRootPage = getArticlesRootPage(pageOwnerId);
+    } else {
+      wiki = createNewsRootPage(WikiType.GROUP.name().toLowerCase(), pageOwnerId);
+      if (wiki != null) {
+        articlesRootPage = getArticlesRootPage(wiki.getOwner());
+      }
+    }
```

Comment:
Here we can have already the wiki (app notes already opened), in that case the articles root page will not be retrieved.
So IMO in order to cover all cases you should:

Test if the articles root page exists, if not we test if the wiki exists, if not we create it (with the homepage), else we create just the news root page